### ISSUE TITLE
Generating `@angular-eslint/schematics:convert-tslint-to-eslint` failed in multiple application project

### ISSUE THREAD

**Issue by Galoitsu** [angular-eslint/angular-eslint#350]
My angular project has multiple application such as `app1` and `app2`.
More importantly, `app1` and `app2` had already migrated from TSLint.
On this condition, I added new application `app3` using `ng generate application app3` and ran command `ng g @angular-eslint/schematics:convert-tslint-to-eslint app3`.
But that failed and show as follows.

```bash
$ ng g @angular-eslint/schematics:convert-tslint-to-eslint app3
Unexpected error: Command failed: npx tslint --print-config "projects/app3/tslint.json"
Failed to load <root>/projects/app3/tslint.json: Invalid "extends" configuration 
value - could not require "../../tslint.json". Review the Node lookup algorithm (https://nodejs.org/api/modules.html#modules_all_together) for the approximate method TSLint uses to find the referenced configuration file.
```

I know that `tslint.json` not exist but I had already deleted `tslint.json` by reference to `angular-eslint/README.md`.
I think that we should be able to migrate from TSLint using `ng g @angular-eslint/schematics:convert-tslint-to-eslint` command even if we add application later.

thanks.

**Comment by JamesHenry** [comment]
Hi @Galoitsu in this scenario you would want to just generate an ESLint project directly.

You can run:

```
ng g @angular-eslint/schematics:app app3
```

...instead of creating one with TSLint first and then converting.

Behind the scenes this schematic invokes the standard Angular CLI app generator.

Hope that helps!
